server/api/index.js

```diff
@@ -68,6 +61,23 @@ apiRouter.post('/message/add', (req, res) => {
     });
 });
 
+// Delete Message
+apiRouter.delete('/message/delete/:id', (req, res) => {
+    const { id } = req.params;
+    const showAll = `select * from messages;`;
```

Comment:
Not needed

Reply:
showAll variable deleted and SQL query changed.